test/functional/test_site/expected/bugs/index.html

```diff
@@ -75,13 +81,14 @@ <h2 id="remove-extra-space-in-links">Remove extra space in links<a class="fa fa-
       <a href="https://github.com">link text</a>]</p>
   </div>
 </div>
-<div id="flex-div"></div>
-<footer>
+            
+        </div>
+        <footer>
   <div class="text-center">
     Default footer
   </div>
 </footer>
-</div>
+    </div>
```

Comment:
The indentation here seems off too

Reply:
Might be due to the indentation within `page.ejs`. Seems like a legacy issue since we have been using 4 space indentation within `page.ejs` but 2 space indentation for `htmlBeautify`.